using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -107,7 +109,9 @@ ACPIdentity.syncIdentifier(identifierType, identifier, ACPMobileVisitorAuthentic
 
 ### syncIdentifiers <a id="syncIdentifiers"></a>
 
-The function of this API is the same as the `syncIdentifier` API. This API passes a list of identifiers, and each identifier contains an `identifier type` as the key and an `identifier` as the value. In each identifier pair, if either the `identifier type` or `identifier` contains a null or an empty string, the identifier is ignored by the Identity extension.
+The function of this API is the same as the `syncIdentifier` API. This API passes a list of identifiers, and each identifier contains an `identifier type` as the key and an `identifier` as the value. In each identifier pair, if the `identifier type` contains a null or an empty string, the identifier is ignored by the Identity extension. 
+
+Starting with _ACPIdentity v2.1.3 (iOS)_ and _Identity v1.1.2 (Android)_, if a previously synced `identifier type` contains a null or empty `identifier` value, it is removed from the Identity shared state, local storage and it is not be synced with the Adobe ECID Service.
```

Comment:
I'm confused by this wording of "if a previously synced ...". It seems to imply that if a previous call to sync contains a null identifier then this call to sync will remove it. Can you use the wording as above and state "if the new identifier value is null or empty ..." ?